mcimx6q-sdb.coffee

```diff
@@ -0,0 +1,34 @@
+deviceTypesCommon = require 'resin-device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+module.exports =
+	slug: 'mcimx6q-sdb'
+	aliases: [ 'mcimx6q-sdb' ]
+	name: 'mcimx6q-sdb'
+	arch: 'armv7hf'
+	state: 'released'
```

Comment:
maybe have this state "experimental" until some tests are conducted on this board